docs/organizer-guide/confs/community-events.rst

```diff
@@ -0,0 +1,20 @@
+Community Events
+================
+
+We are happy to promote events that are created by our community.
+They are **not** officially organized by Write the Docs organizers,
+but they are sister events that share a similar goal.
+
+Requirements
+------------
+
+In order to be listed as a community event,
+we require a few things.
```

Comment:
general: any chance we can decide to use one-sentence-per-line? the line breaks are confusing when editing/reviewing